Common/OJS.Common/GlobalConstants.cs

```diff
@@ -84,6 +85,7 @@ public static class MimeTypes
         public static class HeaderKeys
         {
             public const string ContentDisposition = "Content-Disposition";
+            public const string HostOrigin = "Host-Origin";
```

Comment:
Shouldn't the header be named just "Origin"? https://http.dev/origin

Reply:
Done.